brave/src/main/java/com/linecorp/armeria/server/brave/ArmeriaHttpServerParser.java

```diff
@@ -51,23 +53,27 @@ private ArmeriaHttpServerParser() {
     }
 
     @Override
-    public <T> void request(HttpAdapter<T, ?> rawAdapter, T req, SpanCustomizer customizer) {
-        super.request(rawAdapter, req, customizer);
+    public void parse(brave.http.HttpRequest request, TraceContext context, SpanCustomizer span) {
```

Comment:
nit: `span` -> `customizer` or `spanCustomizer`

Reply:
choosing not to unless you think this is very important

Reply:
ditto on opting out, but if you really really want it to I can change it